src/main/java/net/snowflake/client/core/HttpUtil.java

```diff
@@ -178,6 +176,17 @@ static CloseableHttpClient buildHttpClient(
       OCSPMode ocspMode, File ocspCacheFile, boolean downloadCompressed) {
     // set timeout so that we don't wait forever.
     // Setup the default configuration for all requests on this client
+
+    String ttlSystemProperty = System.getProperty(JDBC_TTL);
```

Comment:
This is unsafe. use `systemGetProperty` method that wraps `System.getProperty`. In some environment `System.getProperty` fails and raises an exception